pkgs/applications/graphics/gscan2pdf/default.nix

```diff
@@ -87,7 +94,7 @@ perlPackages.buildPerlPackage rec {
 
   nativeCheckInputs = [
     imagemagick
-    libtiff
+    libtiff_4_5 # needs older libtiff version. See commit f57a4b0a for more infos
```

Comment:
The reason should be written here. Also using short Shas for long term is not safe.

Reply:
Done